iXBRLViewerPlugin/viewer/src/js/footnote.js

```diff
@@ -21,6 +21,12 @@ export class Footnote {
         return this.textContent();
     }
 
+    readableValueHTML() {
+        const span = document.createElement("span");
+        span.append(document.createTextNode(this.textContent()));
```

Comment:
I guess pre-existing we're losing all formatting (including paragraph breaks).  Should something like [this logic](https://github.com/Arelle/ixbrl-viewer/blob/8fd5ca0211fdccaec5252540111886b9e5d8761d/iXBRLViewerPlugin/viewer/src/js/fact.js#L117-L127) be used in the methods of this class?